e2e/support/bamboo/playwright.Dockerfile

```diff
@@ -0,0 +1,14 @@
+# The image version should match the Playwright version specified in the package.json file
+FROM mcr.microsoft.com/playwright:v1.51.1-jammy
```

Comment:
This version has to match the version used for `@playwright/test` in `package.json`. See the note in [this section](https://playwright.dev/docs/docker#image-tags) of the Playwright docs. We should add something like [this](https://github.com/openmrs/openmrs-esm-patient-chart#updating-playwright) to the README as part of this PR too.